src/styles/main.scss

```diff
@@ -1,3 +1,207 @@
+
+ul {
+  list-style: none;
+  list-style-type: none;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+
+  &__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+  }
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: 60px;
+    font-size: 12px;
+    font-weight: 500;
+    white-space: nowrap;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: #060b35;
+  }
+
+  &__item {
+    margin-left: 20px;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
```

Comment:
Check the 9th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md?rgh-link-date=2022-03-08T19%3A07%3A41Z). Use nesting in all places in your code